README.rst

```diff
@@ -120,6 +127,8 @@ On Android::
     '/data/data/com.termux/cache/SuperApp/log'
     >>> user_config_dir(appname)
     '/data/data/com.termux/shared_prefs/SuperApp'
+    >>> user_documents_dir(appname)
+    '/data/media/0/Documents'
```

Comment:
This path not accessible on non-rooted devices, so it must be changed to `/storage/emulated/0/Documents`